docs/_docs/03-02-product-guide-commenter-features.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Commenter Features
+permalink: /commenter-features/
+---
+
+## Signing up for Talk
+
+There are 2 ways that newsrooms can support signup/login functionality with Talk:
+
+Use Talk’s auth plugin out of the box (supports account registration with username and password, as well as features like forgot password)
+
+Create their own auth plugin (you can use ours as a template) to integrate with your own auth systems
```

Comment:
Our auth plugin won’t work to store state unless they open a pop up. I’ll update this.

Reply:
Ok!

Reply:
Just correct the second paragraph here to read:

> Create their own auth plugin to integrate with your own auth systems

This is pending a link to the "creating a custom auth plugin" section.

Reply:
Fix forthcoming!